Lib/email/utils.py

```diff
@@ -417,8 +417,13 @@ def decode_params(params):
         for name, continuations in rfc2231_params.items():
             value = []
             extended = False
-            # Sort by number
-            continuations.sort()
+            # Sort by number, treating None as 0 if there is no 0, ignore it if there is already a 0.
+            has_zero = any(x[0] == 0 for x in continuations)
+            if has_zero:
+                continuations = [x for x in continuations if x[0] is not None]
+            else:
+                continuations = [(0 if x[0] is None else x[0], x[1], x[2]) for x in continuations]
```

Comment:
```suggestion
                continuations = [(x[0] or 0, x[1], x[2]) for x in continuations]
```

This is a slight shorter way to write it (0 will still be replaced by 0).

Reply:
Got it thanks!